tripal_chado/tests/src/Kernel/ChadoTestKernelBase.php

```diff
@@ -150,4 +150,49 @@ protected function prepareEnvironment(array $functionality) {
       $this->installSchema('tripal', ['tripal_import', 'tripal_jobs']);
     }
   }
+
+  /**
+   * A helper function to add a bundle term reference
+   *
+   * @param \Drupal\tripal\TripalDBX\TripalDbxConnection $chado
+   *   A chado database object.
+   * @param string $table
+   *   The chado table name
+   * @param int $pkey_id
+   *   The primary key value for the specified $table
+   *
+   * @return void
+   */
+  public function addFixedValue($chado, $table, $pkey_id) {
```

Comment:
This method should be added to the ChadoTestTrait so it is included in the functional and kernel environments rather then duplicated. Assuming I haven't missed a difference?

Reply:
I have moved it there with commit 941645496